cmd/notation/sign.go

```diff
@@ -119,7 +106,7 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 		}
 	}
 
-	fmt.Println(desc.Digest)
+	fmt.Println("Sign succeeded for", desc.Digest)
```

Comment:
@FeynmanZhou @yizha1 Is this intended?

Reply:
We can keep the old output for now. There is one issue #303  addressing this output message. We can update it later.

Reply:
I see. Will remove this change for now.